utils.py

```diff
@@ -12,8 +12,12 @@
 
 CLIENT_ID = os.getenv("SPOTIPY_CLIENT_ID")
 CLIENT_SECRET = os.getenv("SPOTIPY_CLIENT_SECRET")
-REDIRECT_URI = os.getenv("REDIRECT_URI")
 
+# Comment the below line while working on your local machine
```

Comment:
Why change back to hard encoded REDIRECT_URI instead of loading from environment variable?

Reply:
I thought it's more work if we load it from .env because you have to also go into travis ci to change the environment variable

Reply:
Got it